### PR TITLE
Improve rewrite section in tactic chapter.

### PR DESCRIPTION
**Pull request by Zimmi48**
**Kind:** documentation fixes

Including a fix to the example given in #7407 and an unrelated fix in a previous mistake of mine in an index.
This is based on #7487 to make it compile.

### CODE REVIEW

File: doc/sphinx/proof-engine/tactics.rst [hunk 1]
```diff
@@ -2634,28 +2625,25 @@ as regular setoids for :tacn:`rewrite` and :tacn:`setoid_replace` (see
...
-.. tacv:: change @term in @ident
-.. tacv:: change @term with @term in @ident
-.. tacv:: change @term at {+ @num} with @term in @ident
+   .. tacv:: change @term {? {? at {+ @num}} with @term} in @ident
 
-  This applies the change tactic not to the goal but to the hypothesis :n:`@ident`.
+      This applies the :tacn:`change` tactic not to the goal but to the hypothesis :n:`@ident`.
 
-See also: :ref:`Performing computations <performingcomputations>`
+   .. seealso :: :ref:`Performing computations <performingcomputations>`
```
Comment (maximedenes) [inline doc/sphinx/proof-engine/tactics.rst]: is the space intended?

Comment (Zimmi48) [reply doc/sphinx/proof-engine/tactics.rst]: After `seealso`? No, oops!

Comment (Zimmi48) [reply doc/sphinx/proof-engine/tactics.rst]: For some reason, it didn't prevent Sphinx from rendering it correctly: https://coq.gitlab.io/-/coq/-/jobs/68113531/artifacts/_install_ci/share/doc/coq/sphinx/html/proof-engine/tactics.html#coq:tacn.change

**Comment by Zimmi48** [comment]
> For some reason, it didn't prevent Sphinx from rendering it correctly: https://coq.gitlab.io/-/coq/-/jobs/68113531/artifacts/_install_ci/share/doc/coq/sphinx/html/proof-engine/tactics.html#coq:tacn.change

I pushed a new version anyways.

**Review by maximedenes — APPROVED** [review]
Had only a superficial look. Looks fine!

**Pull request merged**